db/seeds/certifications.rb

```diff
@@ -0,0 +1,14 @@
+certifications = [
+  "Driving License: Class 2B, 2A, 2",
+  "Driving License: Class 3, 3A",
+  "Driving License: Class 4",
+  "First Aid",
+  "CPR",
+  "Counselling",
+  "Therapy",
+  "Food Hygiene Course"
+]
+
+certifications.each do |certification|
+  Certification.create(name: certification)
```

Comment:
this should be `create!` so when something is wrong, it will stop the seeding by raise an exception